cache/transcoding.go

```diff
@@ -134,7 +137,16 @@ func (c *TranscodingCache) GetAll() map[string]*SegmentInfo {
 
 func (c *TranscodingCache) Store(streamName string, info SegmentInfo) {
 	c.mutex.Lock()
+	defer c.mutex.Unlock()
 	info.updatedAt = time.Now()
 	c.pushes[streamName] = &info
-	c.mutex.Unlock()
+	c.Print()
```

Comment:
We should definitely print information about what's happening (Storing to / removing from the cache) and I don't love this "Debug" flag. Let's just be bold and log it until we hit traffic levels where the logging volume is an issue